extensions/wikia/CollectionView/styles/CollectionView.scss

```diff
@@ -1,62 +1,28 @@
-@import 'skins/shared/color';
-@import "skins/shared/mixins/columns";
+$miniature-size: 100px;
```

Comment:
by item I understand image + caption so let's make it $collection-view-image-size